sdk/search/search-documents/test/utils/setupIndex.ts

```diff
@@ -415,3 +419,275 @@ export async function populateIndex(client: SearchClient<Hotel>): Promise<void>
+            targetName: "locations"
+          }
+        ]
+      }
+    ]
+  },{
+    name: `my-azureblob-skillset-3`,
+    description: `Skillset description`,
+    skills: [
+      {
+        odatatype: "#Microsoft.Skills.Text.EntityRecognitionSkill",
+        inputs: [
+          {
+            name: "text",
+            source: "/document/merged_content"
+          },
+          {
+            name: "languageCode",
+            source: "/document/language"
+          }
+        ],
+        outputs: [
+          {
+            name: "persons",
+            targetName: "people"
+          },
+          {
+            name: "organizations",
+            targetName: "organizations"
+          },
+          {
+            name: "locations",
+            targetName: "locations"
+          }
+        ]
+      }
+    ]
+  },{
+    name: `my-azureblob-skillset-4`,
+    description: `Skillset description`,
+    skills: [
+      {
+        odatatype: "#Microsoft.Skills.Text.EntityRecognitionSkill",
+        inputs: [
+          {
+            name: "text",
+            source: "/document/merged_content"
+          },
+          {
+            name: "languageCode",
+            source: "/document/language"
+          }
+        ],
+        outputs: [
+          {
+            name: "persons",
+            targetName: "people"
+          },
+          {
+            name: "organizations",
+            targetName: "organizations"
+          },
+          {
+            name: "locations",
+            targetName: "locations"
+          }
+        ]
+      }
+    ]
+  },{
+    name: `my-azureblob-skillset-5`,
+    description: `Skillset description`,
+    skills: [
+      {
+        odatatype: "#Microsoft.Skills.Text.EntityRecognitionSkill",
+        inputs: [
+          {
+            name: "text",
+            source: "/document/merged_content"
+          },
+          {
+            name: "languageCode",
+            source: "/document/language"
+          }
+        ],
+        outputs: [
+          {
+            name: "persons",
+            targetName: "people"
+          },
+          {
+            name: "locations",
+            targetName: "locations"
+          }
+        ]
+      }
+    ]
+  }];
+  for(const skillSet of skillSets) {
+    await client.createSkillset(skillSet);
+  }
+}
+
+// eslint-disable-next-line @azure/azure-sdk/ts-use-interface-parameters
+export async function deleteSkillsets(client: SearchIndexerClient): Promise<void> {
+  await client.deleteSkillset("my-azureblob-skillset-1");
+  await client.deleteSkillset("my-azureblob-skillset-2");
+  await client.deleteSkillset("my-azureblob-skillset-3");
+  await client.deleteSkillset("my-azureblob-skillset-4");
+  await client.deleteSkillset("my-azureblob-skillset-5");
+}
+
+// eslint-disable-next-line @azure/azure-sdk/ts-use-interface-parameters
+export async function createIndexers(client: SearchIndexerClient): Promise<void> {
+  const indexers:SearchIndexer[] = [{
```

Comment:
same thought here, let's build this thing up and calculate the new name instead of writing it out

Reply:
Done